src/commcare_cloud/ansible/group_vars/couchdb2_proxy.yml

```diff
@@ -3,4 +3,4 @@ haproxy_username: admin
 haproxy_stats_password: "{{ secrets.HAPROXY_STATS_PASSWORD | default(None) }}"
 haproxy_stats_bind_address: "{{ inventory_hostname }}"
 haproxy_stats_port: 1936
-couchdb2_client_max_body_size: 10M
+couchdb2_client_max_body_size: 30M
```

Comment:
Did you mean to edit this for all environments? If you want to override it for echis, you can do that in `environments/echis/public.yml`

Reply:
I 've changed that to defaullt: `10M`